### PR TITLE
[webportal] Change storage representation

### PR DESCRIPTION
**Pull request by Binyang2014**
As pic

image

### CODE REVIEW

File: src/webportal/src/app/user/fabric/user-profile/storage-list.jsx [hunk 1]
```diff
@@ -36,20 +36,20 @@ function normalizePath(path) {
...
   const data = server.data;
   switch (server.type) {
     case 'nfs':
       return `nfs://${data.address}/${normalizePath(data.rootPath)}`;
     case 'samba':
       return `smb://${data.address}/${normalizePath(data.rootPath)}`;
     case 'azurefile':
-      fileAddress = `${data.accountName}.<AZURE_STORAGE_DNS_SUFFIX>`;
-      return `smb://${fileAddress}/${normalizePath(data.fileShare)}`;
+      return `AccountName: ${data.accountName} Path: ${normalizePath(
```
Comment (abuccts) [inline src/webportal/src/app/user/fabric/user-profile/storage-list.jsx]: pls split container/share name from "path", mounting only needs container/share name

Comment (Binyang2014) [reply src/webportal/src/app/user/fabric/user-profile/storage-list.jsx]: Fixed

File: src/webportal/src/app/user/fabric/user-profile/storage-list.jsx [hunk 2]
```diff
@@ -36,20 +36,16 @@ function normalizePath(path) {
...
     case 'samba':
       return `smb://${data.address}/${normalizePath(data.rootPath)}`;
     case 'azurefile':
-      fileAddress = `${data.accountName}.<AZURE_STORAGE_DNS_SUFFIX>`;
-      return `smb://${fileAddress}/${normalizePath(data.fileShare)}`;
+      return `StorageAccount: ${data.accountName}; FileShare: ${data.fileShare}; Path: `;
     case 'azureblob':
-      blobAddress = `${data.accountName}.<AZURE_STORAGE_DNS_SUFFIX>`;
-      return `http://${blobAddress}/${normalizePath(data.containerName)}`;
+      return `StorageAccount: ${data.accountName}; Container: ${data.accountName}; Path: `;
```
Comment (abuccts) [inline src/webportal/src/app/user/fabric/user-profile/storage-list.jsx]: Container -> BlobContainer, data.accountName -> data.containerName?

Comment (abuccts) [inline src/webportal/src/app/user/fabric/user-profile/storage-list.jsx]: could you make the title bold by returning `<b>Storage Account:</b> ...`?

Comment (Binyang2014) [reply src/webportal/src/app/user/fabric/user-profile/storage-list.jsx]: Fix containerName issue. Still use Container, same as Azure

Comment (Binyang2014) [reply src/webportal/src/app/user/fabric/user-profile/storage-list.jsx]: Fixed

**Pull request merged**